packages/itwinui-css/src/slider/slider.scss

```diff
@@ -82,8 +64,6 @@ $tick-height: $iui-baseline; // 11px
 @mixin iui-slider-track {
   pointer-events: none;
   position: absolute;
-  height: $track-height;
-  left: 0;
   top: $iui-sm;
```

Comment:
If think this one is also horizontal specific.

Reply:
**[DONE]** I think this code is from an old commit. But thanks, yes, I reconfirmed that it is now under `iui-slider-horizontal`